v4/export/writer_util.go

```diff
@@ -12,6 +12,8 @@ import (
 	"sync"
 	"time"
 
+	"golang.org/x/time/rate"
```

Comment:
Combine this line with upper dependencies?

Reply:
resolved.